tv_shows_app/src/components/features/reviews/StarReview.tsx

```diff
@@ -0,0 +1,30 @@
+import { Container, Flex, Image} from "@chakra-ui/react";
+
+interface IStarReview {
+    noOfStars: number
+}
+
+
+export const StarReview = ({noOfStars}: IStarReview) => {
+    let tempList = [];
+    for(let i = 1; i <= 5; i++) {
+        tempList.push(
+        <Image 
+            src = {(i <= noOfStars) ? "/filled-star.png" : "/empty-star.png"} 
+            fallbackSrc={"https://fakeimg.pl/600x400/ff0000/ffffff?text=Nema+slike+:("}
+            alt = "Nema slike"
+            width={"15%"}
+            key={"slika-" + i}/>
+        );
+    }
+    return <Flex
+        flexDirection={"row"}
+        gap={"0"}
+        width={"30%"}
+        marginTop={"1"}
+        marginBottom={"3"}
+
+        >
+        {tempList}
+     </Flex>
```

Comment:
Ovo smo prošli skupa na konzultacijama.
Koristili smo StarIcon komponentu od chakre, proslijedili smo ovdje value (preko kojeg smo znali koja je zvijezdica odabrana -> to je bio rating) i onChange event na klik zvijezdice gdje smo parentu rekli koju smo zvijezdicu kliknuli.
Onda je parent drzao neki state o tome koji je rating odabran (forma)
Ako se nesjecas, javis se pa prođemo opet skupa